db/setup/queries.sql

```diff
@@ -31,5 +31,50 @@ VALUES
   (
     'fportela-admin',
     'fportela-admin@transcendence.com',
-    'Fiz Portela Savariz'
+    'Fiz Purple'
+  );
+
+INSERT INTO
+  Users ("username", "email", "fullName")
+VALUES
+  (
+    'awesome-user',
+    'awesome-user@transcendence.com',
+    'Awesome Gray'
+  );
```

Comment:
Added some more mock users. As agree with gbudau, we could populate our app using https://github.com/42AGV/ft_transcendence/blob/14088c6dbced78593519d09a4af6b3f17d4e2c34/transcendence-app/seeds/seed_users.ts

Reply:
Sorry, I think it was me who put that information. I will be more careful in the future.
But Mr. Purple sounds great for me